packages/capnp-rpc-unix/capnp-rpc-unix.1.2.3/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+synopsis:
+  "Cap'n Proto is a capability-based RPC system with bindings for many languages"
+description:
+  "This package contains some helpers for use with traditional (non-Unikernel) operating systems."
+maintainer: "Thomas Leonard <talex5@gmail.com>"
+authors: "Thomas Leonard <talex5@gmail.com>"
+license: "Apache-2.0"
+homepage: "https://github.com/mirage/capnp-rpc"
+doc: "https://mirage.github.io/capnp-rpc/"
+bug-reports: "https://github.com/mirage/capnp-rpc/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "capnp-rpc-net" {= version}
+  "cmdliner" {>= "1.1.0"}
+  "cstruct-lwt"
+  "astring"
+  "fmt" {>= "0.8.7"}
+  "logs"
+  "extunix"
+  "base64" {>= "3.0.0"}
+  "dune" {>= "3.0"}
+  "alcotest" {>= "1.6.0" & with-test}
+  "alcotest-lwt" { >= "1.6.0" & with-test}
+  "mirage-crypto-rng-lwt" {>= "0.11.0"}
+  "mdx" {>= "2.2.1" & with-test}
+  "lwt" {>= "5.6.1"}
+  "asetmap" {with-test}
+]
+conflicts: [
+  "jbuilder" {with-test}
+]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+dev-repo: "git+https://github.com/mirage/capnp-rpc.git"
+url {
+  src:
+    "https://github.com/mirage/capnp-rpc/releases/download/v1.2.3/capnp-rpc-1.2.3.tbz"
+  checksum: [
+    "sha256=828002d67b9591d1645266c504e3fabc66b750229244a68b0a846c3c93f73715"
+    "sha512=c29f13ada74f3f8c80aa591f0fad60801ea72aea6aaa5299b2edee08e080061c5ac054392678ed8910962b1348f1e61790ce30febfc391ddb8c5ac01d56f3160"
+  ]
+}
+x-commit-hash: "86427f68fa9a851fad6317cfda5e8b596add7fe9"
+x-ci-accept-failures: ["macos-homebrew"]
```

Comment:
Only the tests fail so only the tests should be disabled. `x-ci-accept-failures` is not made for this
```suggestion
```

Reply:
I guess. Ideally we only want to disable the tests in opam-repo-ci, not on macos in general.

Reply:
not really. As long as you run them using opam they will fail on macOS so constraining them in the opam file is the correct way to do it.